src/main/java/edu/harvard/iq/dataverse/engine/command/impl/AbstractDatasetCommand.java

```diff
@@ -155,32 +155,30 @@ protected void tidyUpFields(DatasetVersion dsv) {
      * @param ctxt
      * @throws CommandException
      */
-    protected void registerExternalIdentifier(Dataset theDataset, CommandContext ctxt) throws CommandException {
+    protected void registerExternalIdentifier(Dataset theDataset, CommandContext ctxt, boolean retry) throws CommandException {
         if (!theDataset.isIdentifierRegistered()) {
             GlobalIdServiceBean globalIdServiceBean = GlobalIdServiceBean.getBean(theDataset.getProtocol(), ctxt);
             if ( globalIdServiceBean != null ) {
                 if (globalIdServiceBean instanceof FakePidProviderServiceBean) {
-                    try {
-                        globalIdServiceBean.createIdentifier(theDataset);
-                    } catch (Throwable ex) {
-                        logger.warning("Problem running createIdentifier for FakePidProvider: " + ex);
-                    }
-                    theDataset.setGlobalIdCreateTime(getTimestamp());
-                    theDataset.setIdentifierRegistered(true);
-                    return;
```

Comment:
I think this was just to avoid the retry attempts - same set of calls otherwise.